src/gui/owncloudgui.cpp

```diff
@@ -89,6 +89,10 @@ ownCloudGui::ownCloudGui(Application *parent)
 
     connect(_tray.data(), &Systray::shutdown,
         this, &ownCloudGui::slotShutdown);
+    connect(_tray.data(), &Systray::openShareDialog,
+        this, [=](const QString &sharePath, const QString &localPath) {
+                slotShowShareDialog(sharePath,localPath, ShareDialogStartPage::UsersAndGroups);
```

Comment:
Missing space after the first comma

Man... I wish we had a bot checking the formatting so that I don't do that myself. Actually the line just before is wrongly indented as well, but I guess you hate me enough like this already. ;-)